packages/flutter/lib/src/widgets/scroll_activity.dart

```diff
@@ -483,7 +486,7 @@ class DragScrollActivity extends ScrollActivity {
   }
 
   @override
-  bool get shouldIgnorePointer => true;
+  bool get shouldIgnorePointer => _controller?._kind != PointerDeviceKind.trackpad;
```

Comment:
I think the problem can be solved by allowing only down events. Should we allow hover events? I'm a little concerned about performance, or any reasons why we decided to ignore these events before.